docs/en/wallet-instance-attestation.rst

```diff
@@ -2,56 +2,676 @@
+|                                   | wallet provider.                  |
++-----------------------------------+-----------------------------------+
+| grant_types_supported             | The type of grants supported by   |
+|                                   | the endpoint token. Therefore,    |
+|                                   | for the wallet provider the token |
+|                                   | is equivalent only to the wallet  |
+|                                   | instance attestation, therefore   |
+|                                   | this attribute will contain an    |
+|                                   | array with only one element:      |
+|                                   | urn:ietf:params:oauth:grant-type: |
+|                                   | wallet-instance-attestation       |
++-----------------------------------+-----------------------------------+
+| token_endpoint_auth_methods_suppo | Supported authentication method   |
+| rted                              | for the endpoint token. In our    |
+|                                   | case it's just private_key_jwt    |
++-----------------------------------+-----------------------------------+
+| token_endpoint_auth_signing_alg_v | List of supported signature       |
+| alues_supported                   | algorithms                        |
++-----------------------------------+-----------------------------------+
+
+Payload `federation_entity`
+'''''''''''''''''''''''''''
++-------------------+----------------------------------------+
+| **key**           | **value**                              |
++-------------------+----------------------------------------+
+| organization_name | Organization name                      |
++-------------------+----------------------------------------+
+| homepage_uri      | Organization website                   |
++-------------------+----------------------------------------+
+| tos_uri           | Url to the terms of use                |
++-------------------+----------------------------------------+
+| policy_uri        | Url to the privacy policy              |
++-------------------+----------------------------------------+
+| logo_uri          | URL of the organization logo           |
++-------------------+----------------------------------------+
+
+Non-normative example
+^^^^^^^^^^^^^^^^^^^^^
+
+.. code-block:: javascript
+
+  {
+    "alg": "ES256",
+    "kid": "5t5YYpBhN-EgIEEI5iUzr6r0MR02LnVQ0OmekmNKcjY",
+    "typ": "entity-statement+jwt"
+  }
+  .
+  {
+    "iss": "https://wallet.italia.it",
+    "sub": "https://wallet.italia.it",
+    "jwk": {
+      "keys": [
+        {
+          "crv": "P-256",
+          "kty": "EC",
+          "x": "qrJrj3Af_B57sbOIRrcBM7br7wOc8ynj7lHFPTeffUk",
+          "y": "1H0cWDyGgvU8w-kPKU_xycOCUNT2o0bwslIQtnPU6iM",
+          "kid": "5t5YYpBhN-EgIEEI5iUzr6r0MR02LnVQ0OmekmNKcjY"
+        }
+      ]
+    },
+    "metadata": {
+      "eudi_wallet_provider": {
```

Comment:
@grausof 

what about renaming it to openid_wallet_provider?
or oauth_wallet_provider?